packages/framework/fluid-framework/api-extractor.json

```diff
@@ -1,4 +1,11 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "extends": "@fluidframework/build-common/api-extractor-common-strict.json"
+  "extends": "@fluidframework/build-common/api-extractor-common-report.json",
```

Comment:
Note: this change is required until all the packages we are bundling here also conform to the strict config.